doc/guides/backporting-to-release-lines.md

```diff
@@ -68,25 +66,24 @@ hint: with 'git add <paths>' or 'git rm <paths>'
 hint: and commit the result with 'git commit'
 ```
 
-* Make the required changes to remove the conflicts, add the files to the index
-  using `git add`, and then commit the changes. That can be done with
-  `git cherry-pick --continue`.
-* Leave the commit message as is. If you think it should be modified, comment
-  in the Pull Request.
-* Make sure `make -j4 test` passes
-* Push the changes to your fork and open a pull request.
-* Be sure to target the `v7.x-staging` branch in the pull request.
-
-### Helpful Hints
-
-* Please include the backport target in the pull request title in the following
-  format: `(v7.x backport) <commit title>`
-  * Ex. `(v4.x backport) process: improve performance of nextTick`
-* Please check the checkbox labelled "Allow edits from maintainers".
-  This is the easiest way to to avoid constant rebases.
-
-In the event the backport pull request is different than the original,
-the backport pull request should be reviewed the same way a new pull request
-is reviewed.
+5. Make the required changes to remove the conflicts, add the files to the index
+   using `git add`, and then commit the changes. That can be done with
+   `git cherry-pick --continue`.
+6. Leave the commit message as is. If you think it should be modified, comment
+   in the Pull Request.
+7. Make sure `make -j4 test` passes.
+8. Push the changes to your fork
+9. Open a pull request:
+   1. Be sure to target the `v7.x-staging` branch in the pull request.
```

Comment:
This is maybe just me, but I feel like `a.` `b.` `c.` is more natural than another set of numbers.